app/views/spaces/_branding.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="mt-3 gap-y-3">
+  <%= render CardComponent.new do %>
+    <h3>Branding</h3>
```

Comment:
Maybe "Colors" or "Styling", instead of "Branding"?

"Branding" feels very oriented towards more commercial users, and also not as specific as the functionality that this includes right now.

Reply:
lol maybe "Vibes"